### PR TITLE
Replace `sdb_fmt()` with `rz_strf()`.

### PR DESCRIPTION
**Pull request by Rot127**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

...

**Test plan**

All green

**Closing issues**

...

### CODE REVIEW

**Review by wargio — APPROVED** [review]

File: librz/core/agraph.c [hunk 1]
```diff
@@ -2731,15 +2735,15 @@ static void agraph_set_layout(RzAGraph *g) {
...
-		k = sdb_fmt("agraph.nodes.%s.x", a->title);
-		sdb_num_set(g->db, k, rebase(g, a->x), 0);
-		k = sdb_fmt("agraph.nodes.%s.y", a->title);
-		sdb_num_set(g->db, k, rebase(g, a->y), 0);
-		k = sdb_fmt("agraph.nodes.%s.w", a->title);
-		sdb_num_set(g->db, k, a->w, 0);
-		k = sdb_fmt("agraph.nodes.%s.h", a->title);
-		sdb_num_set(g->db, k, a->h, 0);
+		char buf[1024] = { 0 };
+		rz_strf(buf, "agraph.nodes.%s.x", a->title);
```
Comment (wargio) [inline librz/core/agraph.c:2739]: `agraph.nodes..w` is 15 chars long, if we count also `\0` then we have 16 bytes.
probably `%s` should be limited to `1000` chars.

Comment (wargio) [reply librz/core/agraph.c:2739]: the same for other stuff, but i guess doesn't matter, since `sdb_fmt` was limited to 512 bytes.

Comment (Rot127) [reply librz/core/agraph.c:2739]: Just saw that `x->title` is at some point truncated to 255 bytes. So lets limit it to 512?

Comment (wargio) [reply librz/core/agraph.c:2739]: then yes you can move the buffer down.

**Review by wargio — APPROVED** [review]

**Pull request merged**